libs/slam/async_slam/async_slam.cpp

```
@@ -171,17 +166,18 @@ void AsyncSlam::TrackResult(FrameId frameId, int64_t timestamp_ns, const odom::I
           continue;  // remove landmarks outside the max distance
         }
 
-        if (x.cam_id >= images.size() || images[x.cam_id] == nullptr) {
+        if (cam_id >= images.size() || images[cam_id] == nullptr) {
           continue;
         }
-        if (x.cam_id >= rig_.num_cameras) {
+        if (cam_id >= rig_.num_cameras) {
```

Cppcheck (2.21.0)

[style] The function 'StringToDistortionModel' is never used.

(unusedFunction)